src/Components/App/App.tsx

```diff
@@ -1,43 +1,31 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { Routes, Route, useLocation, useNavigate } from 'react-router-dom'
+import { Routes, Route, useLocation } from 'react-router-dom'
 import Navbar from '../Navbar/Navbar';
 import Login from '../Login/Login';
 import DashboardMenu from '../DashboardMenu/DashboardMenu'
-import { useQuery, gql } from '@apollo/client';
 import PendingRequests from '../PendingRequests/PendingRequests';
 import Home from '../Home/Home';
 import BookDetails from '../BookDetails/BookDetails';
 import MyBooks from '../MyBooks/MyBooks';
 import BorrowedBooks from '../BorrowedBooks/BorrowedBooks';
-import { User } from '../../Interfaces'
+import { currentUser, User, Location } from '../../Interfaces'
 import AddBook from '../AddBook/AddBook';
 import { PageNotFound } from '../PageNotFound/PageNotFound';
 
-
-interface Location {
```

Comment:
Thanks for cleaning all of these up! Looks so much cleaners with them in their own file